### DISCUSSION TITLE
Ordering of theme color maps and color vars

### DISCUSSION THREAD

**Discussion by Kopyov** [twbs/bootstrap#34228] [General]
Is there any particular reason why color variables come after $theme-colors map and not before? This makes compilation impossible if i want to add any of custom color variables in the map, e.g. if i add $blue-100 to $theme-colors map, it won't compile.

It works with custom map with merging, but it also needs to be included after all color variables.

**Comment by mdo** [comment]
https://getbootstrap.com/docs/5.0/customize/sass/#maps-and-loops shows how to modify the maps. Is this not working for you?
